test.js

```diff
@@ -1,26 +1,26 @@
 import test from 'ava';
-import dotProp from './';
+import m from './';
```

Comment:
It's the new @sindresorhus convention ;). At first, it was `fn`, but `fn` isn't general enough.